src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const url
+= 'https://mate-academy.github.io/phone-catalogue-static/api/phones.json';
```

Comment:
you can add this comment to fix such problems
```suggestion
// eslint-disable-next-line
const url = 'https://mate-academy.github.io/phone-catalogue-static/api/phones.json';
```